src/library_dylink.js

```diff
@@ -204,12 +210,16 @@ var LibraryDylink = {
 #endif
         if (typeof value == 'function') {
           /** @suppress {checkTypes} */
-          GOT[symName].value = addFunction(value, value.sig);
+          GOT[symName].value = {{{ to64('addFunction(value, value.sig)') }}};
 #if DYLINK_DEBUG
           err('assigning table entry for : ' + symName + ' -> ' + GOT[symName].value);
 #endif
         } else if (typeof value == 'number') {
+          GOT[symName].value = {{{ to64('value') }}};
```

Comment:
why is `to64` needed here?

Reply:
In wasm64 the GOT contains pointers/addresses.  These address are all wasm globals of type i64.   Assigning to the GOT requires a bigint under wasm64 and an number under wasm32, which is exactly what to64 does for us.

In wasm64 mode we deal number numbers almost everwhere, except in place on the boundary like this.